hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -399,6 +432,28 @@ private LookupKeyResponse lookupKey(LookupKeyRequest request,
     return resp.build();
   }
 
+  @RequestFeatureValidator(
+      conditions = ValidationCondition.OLDER_CLIENT_REQUESTS,
+      processingPhase = RequestProcessingPhase.POST_PROCESS,
+      requestType = Type.LookupKey
+  )
+  public static OMResponse disallowLookupKeyResponseWithECReplicationConfig(
+      OMRequest req, OMResponse resp, ValidationContext ctx)
+      throws ServiceException {
+    if (!resp.hasLookupKeyResponse()) {
+      return resp;
+    }
+    if (resp.getLookupKeyResponse().getKeyInfo().hasEcReplicationConfig()) {
```

Comment:
Same as above, the client's actual version matters as much as the relative version.

Reply:
agree, see my answer to @errose28 @kerneltime. As I think we should come up with the refactor prior to 1.3.0 release and that should contain a functionality like this for sure, this changeset is now about expediting the branch merge for the EC branch by ensuring the compatibility parts of it.